src/modules/zig.rs

```diff
@@ -38,7 +37,10 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
                 _ => None,
             })
             .map(|variable| match variable {
-                "version" => Some(Ok(zig_version.clone())),
+                "version" => {
+                    let zig_version = format!("v{}", zig_version_output);
```

Comment:
I think you also need to move the `exec_cmd ` command from line 21 to here.

Reply:
True. Done!

Also fixed the clippy complaint.